yabt/builders/golang.py

```diff
@@ -214,12 +214,21 @@ def go_builder_internal(build_context, target, command, is_binary=True):
     buildenv_workspace = build_context.conf.host_to_buildenv_path(
         workspace_dir
     )
+    target_sources = target.props.sources
+    files_to_link = list(target.props.sources)
```

Comment:
So the sources of the test target include all the test code as well as the application code?

Reply:
the test code is the application code, or I didn't understand what you mean. In GoTest the application is the test. To be more readable, the best thing is to copy the sources to src dir, regardless if this is a GoTest, GoProg or GoPackage. It should work for all.

Reply:
Changed to copy direct sources to 'src/' under workspace.